code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -131,7 +131,7 @@ Des: Removes all infected images from the alien.
 	qdel(src)
 
 /mob/living/carbon/alien/can_hold_items(obj/item/I)
-	return ((I && istype(I, /obj/item/clothing/mask/facehugger)) || (ISADVANCEDTOOLUSER(src) && ..()))
+	return (I && (I.item_flags & XENOMORPH_HOLDABLE || ISADVANCEDTOOLUSER(src)) && ..())
```

Comment:
does this change change the parenthesis logic of this statement (phrase?)?

Reply:
yes

Reply:
lemme make sure I'm getting this right, we went from `if (it exists and it's a facehugger) or (if we're an  advanced tool user and parent is ok with it)` to `if it exists, (and if we're allowed to use it, or if we're an advanced tool user), and parent is ok with it` Could you explain the logic behind your change?

Reply:
why should we ignore what the parent does if it does have that flag? seems like it could be fucky

Reply:
In theory because it does more then just check to see if the mob has the available slots, but that does indeed appear to be the only thing it does. Makes me wonder how this worked before honestly